docs/cudf/source/index.rst

```diff
@@ -14,7 +14,6 @@ the details of CUDA programming.
    :caption: Contents:
```

Comment:
Do we plan to convert this file and the API docs to MyST Markdown as well? I would support that (in a later PR) if you agree.

(It will be a tiny bit annoying to write rST Python docstrings and MyST everywhere else, but I feel comfortable in both languages so it's not an issue for me.)